resources/helm/tools/generate-json-schema.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
```

Comment:
My suggestion (and general practice) is to link to the original source in a comment in the file, and then immediately work on extracting it out into a package that can be re-used (that's how we got chartpress). Otherwise, the files will diverge quickly and run into maintenance related problems.

Reply:
Thanks @yuvipanda! I've added a commit about the origin of the script and I think we can consider this as resolved for now. I'll work to make these scripts accessible via a standalone package in one way or another soon.

Reply:
Happy with the plan here. Thanks for weighing in @yuvipanda.